### PR TITLE
feat: add LinkAce template

### PR DESCRIPTION
**Pull request by peterambrus3**
https://www.linkace.org/

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]

File: web/crux/templates/link-ace.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        },
+        "name": "linkace-app",
+        "ports": [
+          {
+            "external": 6780,
+            "internal": 80
+          }
+        ],
+        "environment": [
+          {
```
Comment (Kovah) [inline web/crux/templates/link-ace.json:81]: The app key is an important secret used to encrypt/decrypt data in LinkAce. It should probably added as a secret then.

Comment (polaroi8d) [reply web/crux/templates/link-ace.json:81]: It's just a template (example), users have to change it when they use their own instance.

Comment (polaroi8d) [reply web/crux/templates/link-ace.json:81]: But yea, a good point we'll move this key to the secret.

Comment (peterambrus3) [reply web/crux/templates/link-ace.json:81]: Thanks for your review, APP_KEY is a secret from now. 👍

Comment (Kovah) [reply web/crux/templates/link-ace.json:81]: Thanks for taking the time to create the template!

**Review by balanceee — APPROVED** [review]

**Pull request merged**